examples/tutorials/research/02_advanced_usage_of_research.ipynb

```diff
@@ -0,0 +1,1005 @@
+{
```

Comment:
<p>Extend the description as you define not only a dataset but a pipeline as well</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/337/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>